src/backend/api/src/proposal_review.rs

```diff
@@ -0,0 +1,46 @@
+use candid::{CandidType, Deserialize};
+
+#[derive(Debug, CandidType, Deserialize, Clone, PartialEq, Eq)]
+pub enum ProposalReviewStatus {
+    #[serde(rename = "draft")]
+    Draft,
+    #[serde(rename = "published")]
+    Published,
+}
+
+#[derive(Debug, CandidType, Deserialize, Clone, PartialEq, Eq)]
+pub struct ProposalReview {
+    pub proposal_id: String,
+    pub user_id: String,
+    pub created_at: String,
+    pub status: ProposalReviewStatus,
+    pub summary: String,
+    pub review_duration_mins: u16,
+    pub build_reproduced: bool,
+    pub reproduced_build_image_id: Option<String>,
+}
+
+#[derive(Debug, Clone, CandidType, Deserialize, PartialEq, Eq)]
+pub struct CreateProposalReviewRequest {
+    pub proposal_id: String,
+    pub summary: Option<String>,
+    pub review_duration_mins: Option<u16>,
+    pub build_reproduced: Option<bool>,
+    pub reproduced_build_image_id: Option<String>,
+}
+
+#[derive(Debug, Clone, CandidType, Deserialize, PartialEq, Eq)]
+pub struct CreateProposalReviewResponse {
+    pub id: String,
```

Comment:
Maybe this could be named `proposal_id` for clarity

Reply:
Do you mean `proposal_review_id`? The `proposal_id` is already present in the `proposal_review` field

Reply:
Nevermind, I misread this as the request, not response. This is fine as-is 👍